### PR TITLE
Add the story of how this codebase came to be

### PR DESCRIPTION
**Pull request by garious**
#### Problem

When folks first come across us or our project, some of the first questions are: "why do we need another blockchain?", "how is yours different?", "what got you to pull the trigger on starting a company?" and "why the name Solana?"

#### Summary of Changes

Consolidating those answers turned out to be a nice intro for the markdown book.

### CODE REVIEW

File: src/introduction.md [hunk 1]
```diff
@@ -3,37 +3,117 @@
...
 author's best effort. It is up to the reader to check and validate their
-accuracy and truthfulness. Furthermore nothing in this project constitutes a
+accuracy and truthfulness. Furthermore, nothing in this project constitutes a
 solicitation for investment.
 
-# Introduction
+# What is Solana?
 
-This document defines the architecture of Solana, a blockchain built from the
+Solana is the name of an open source project that is implementing a new
```
Comment (rob-solana) [inline src/introduction.md]: recommend striking "new" and changing to active voice: "that implements a blockchain"

File: src/introduction.md [hunk 2]
```diff
@@ -3,37 +3,117 @@
...
+
+# About this book
+
+This book defines the architecture of Solana, a blockchain built from the
 ground up for scale. The goal of the architecture is to demonstrate there
 exists a set of software algorithms that in combination, removes software as a
 performance bottleneck, allowing transaction throughput to scale proportionally
 with network bandwidth. The architecture goes on to satisfy all three desirable
-properties of a proper blockchain, that it not only be scalable, but that it is
+properties of a proper blockchain: that it not only be scalable, but that it is
```
Comment (rob-solana) [inline src/introduction.md]: recommend "but that it be" to enhance parallelism

File: src/introduction.md [hunk 3]
```diff
@@ -3,37 +3,117 @@
...
+people were confused if they were the same project. The Loom team decided it
+would rebrand. They chose the name Solana, a nod to a small beach town North of
+San Diego called Solana Beach, where Anatoly, Greg and Stephen lived and surfed
+for three years when they worked for Qualcomm. On March 28th, the team created
+the Solana Labs GitHub organization and renamed Greg's prototype Silk to
+Solana.
+
+In June of 2018, Solana made its first three hires: former Qualcomm Senior
+Directors Michael Vines and Rob Walker, and Principal Engineer Pankaj Garg.
+The team scaled up the technology to run on cloud-based networks and on July
```
Comment (mvines) [inline src/introduction.md]: IMO strike the previous sentence and start this paragraph with “In June of 2018, the team scaled up the tech...”, then remove the “four additional engineering hires...” from the next paragraph.   That is, call out the founding team by name but after that it’s just “the team” regardless of name, company of origin, or discipline (eng, bd, ops, etc)

Comment (garious) [reply src/introduction.md]: Updated, thanks!

**Pull request merged**